interface/wx/toolbar.h

```diff
@@ -122,9 +123,21 @@ class wxToolBarToolBase : public wxObject
 
     wxObject *GetClientData() const;
 
+    virtual bool Enable(bool enable);
```

Comment:
We don't do this consistently, but I'd prefer to avoid documenting the methods that are not meant to be overridden in the user code as `virtual`. It's just an implementation detail in this case.